### PR TITLE
Process - Start and Stopping #Solves 409

### PR DESCRIPTION
**Pull request by danloa**
- The deactivate/activate icons were changed to show tha active/inactive state
- View changed to send the deactivate/activate request depending on the current status
- The +Request button shows only the active processes.
Ref: #409

### CODE REVIEW

File: ProcessMaker/Http/Controllers/Api/Requests/RequestsController.php [hunk 1]
```diff
@@ -40,4 +44,70 @@ public function index(Request $request)
         // Return fractal representation of paged data
         return fractal($requests, new ApplicationTransformer())->respond();
     }
+
+    /**
+     * Returns the list of processes that can be started by the connected user
```
Comment (marcoAntonioNina) [inline ProcessMaker/Http/Controllers/Api/Requests/RequestsController.php]: Where is it verified that users can create Requests?

File: resources/assets/js/components/requests/modal.vue [hunk 2]
```diff
@@ -79,7 +79,7 @@ export default {
       // Now call our api
       // Maximum number of requests returned is 200 but should be enough
       // @todo Determine if we need to paginate or lazy scroll if someone has more than 200 requests
-      window.ProcessMaker.apiClient.get('processes?filter=' + this.filter + '&per_page=200')
+      window.ProcessMaker.apiClient.get('user/processes?filter=' + this.filter + '&per_page=200')
```
Comment (marcoAntonioNina) [inline resources/assets/js/components/requests/modal.vue]: change per_page by default

File: resources/assets/js/processes/components/ProcessesListing.vue [hunk 3]
```diff
@@ -85,10 +85,36 @@
...
+                    ProcessMaker.apiClient
+                        .put('/processes/' + data.uid, {
+                            status: (data.status === 'ACTIVE') ? 'INACTIVE' : 'ACTIVE'
+                        })
+                        .then(response => {
+                            this.loading = false;
+                            document.location.reload();
+                        })
+                        .catch(error => {
+                            // Undefined behavior currently, show modal?
```
Comment (marcoAntonioNina) [inline resources/assets/js/processes/components/ProcessesListing.vue]: you could use it 

ProcessMaker.alert(message, type)

to inform you that the state could not be changed.

File: ProcessMaker/Http/Controllers/Api/Requests/RequestsController.php (hunk 1 above)
Comment (danloa) [reply ProcessMaker/Http/Controllers/Api/Requests/RequestsController.php]: Here we have the problem that we aren't managing assignment rules yet. So for the moment we don't have the documentation. @velkymx  I suggest to create a new ticket to implement this.

File: resources/assets/js/processes/components/ProcessesListing.vue (hunk 3 above)
Comment (velkymx) [reply resources/assets/js/processes/components/ProcessesListing.vue]: We're already catching the error. We should not be including empty methods.

Comment (caleeli) [reply resources/assets/js/processes/components/ProcessesListing.vue]: With #399 the api errors will be shown by default with an alert.

Comment (danloa) [reply resources/assets/js/processes/components/ProcessesListing.vue]: The catch section of the api calls was removed

**Pull request merged**